### PR TITLE
feat(DENG-1360): added queries for more Airflow metadata tables

### PR DESCRIPTION
**Pull request by kik-kik**
# feat(DENG-1360): added queries for more Airflow metadata tables

depends on: https://github.com/mozilla/bigquery-etl/pull/4171

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-1376)

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/monitoring_derived/airflow_dag_tag_v1/init.sql
Comment (lelilia) [inline sql/moz-fx-data-shared-prod/monitoring_derived/airflow_dag_tag_v1/init.sql:1]: Why do you make init files? I thought we moved away from int.sql and you already have a schema.yaml

**Review by lelilia — CHANGES_REQUESTED** [review]

File: sql/moz-fx-data-shared-prod/monitoring_derived/airflow_dag_owner_attributes_v1/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+friendly_name: Airflow DAG owner attributes
+description: List of DAG owner attributes
+labels:
+  incremental: false
+owners:
+  - kignasiak@mozilla.com
```
[the remark this answers is not in the document]
Comment (kik-kik) [reply sql/moz-fx-data-shared-prod/monitoring_derived/airflow_dag_owner_attributes_v1/metadata.yaml:6]: 😬 yes, you are correct.

File: sql/moz-fx-data-shared-prod/monitoring_derived/airflow_dag_tag_v1/init.sql
Comment (kik-kik) [reply sql/moz-fx-data-shared-prod/monitoring_derived/airflow_dag_tag_v1/init.sql:1]: that's a mistake, they should not be there.

**Review by lelilia — APPROVED** [review]
lgtm now

**Pull request merged**